src/components/members.component.js

```diff
@@ -109,38 +144,29 @@ const MembersListComponent = ({
               );
             }}
             underlayColor="transparent"
-            style={[
-              styles.avatarContainer,
-              { marginRight: index < members.length - 1 ? 5 : 0 },
-            ]}
+            style={{ marginRight: index < members.length - 1 ? 5 : 0 }}
           >
-            <Image
-              style={styles.avatar}
+            <Avatar
               source={{
                 uri: item.avatar_url,
               }}
             />
-          </TouchableHighlight>
+          </AvatarContainer>
         )}
         keyExtractor={item => item.id}
-        horizontal
       />
 
-      <LinearGradient
-        style={[styles.scrollGradient, { left: 0 }]}
-        colors={['white', 'rgba(255, 255, 255, 0)']}
-        start={{ x: 0, y: 0.5 }}
-        end={{ x: 1, y: 0.5 }}
+      <ScrollGradient
+        gradientPosition={{ left: 0 }}
+        gradientColors={['white', 'rgba(255, 255, 255, 0)']}
       />
 
-      <LinearGradient
-        style={[styles.scrollGradient, { right: 0 }]}
-        colors={['rgba(255, 255, 255, 0)', 'white']}
-        start={{ x: 0, y: 0.5 }}
-        end={{ x: 1, y: 0.5 }}
+      <ScrollGradient
+        gradientPosition={{ right: 0 }}
+        gradientColors={['rgba(255, 255, 255, 0)', 'white']}
```

Comment:
we could omit the `gradient` prefix from prop names, as the component name is Scroll**Gradient** ;)